bin/refresh-styles

```diff
@@ -4,11 +4,12 @@
 
 source bin/lib.sh
 
+# Pass --minify to this script to minify the tailwind.css file in dev
```

Comment:
suggestion (blocking): mention the `--minify` flag in the docstring on L3

Reply:
Line 3